CodeAcross/index.html

```diff
@@ -168,10 +162,10 @@ <h2 class="date">February 21 2015</h2>
 
             <div class="border">
                 <div class="gallery">
-                    <img src="../img/CfA_logo_lg.png" width="250" />
-                    <img src="../img/rally-logo.png" width="250" />
-                    <img src="../img/galvanize.png" width="250"/>
-                    <img src="../img/you-here.png" width="250" />
+                    <a href="http://www.codeforamerica.org/"><img src="../img/CfA_logo_lg.png" width="250" /></a>
+                    <a href="https://www.rallydev.com/"><img src="../img/rally-logo.png" width="250" /></a>
+                    <a href="http://www.galvanize.it/"><img src="../img/galvanize.png" width="250"/></a>
+                    <a href="#"><img src="../img/you-here.png" width="250" /></a>
```

Comment:
One of the guys at the last hack night mentioned it - thought it was a good idea- for those who are not familiar with CfA 



—
Sent from Mailbox

On Tue, Feb 17, 2015 at 7:21 PM, Willy Douglas <notifications@github.com>
wrote:

>> @@ -168,10 +162,10 @@ <h2 class="date">February 21 2015</h2>
>>  
>>              <div class="border">
>>                  <div class="gallery">
>> -                    <img src="../img/CfA_logo_lg.png" width="250" />
>> -                    <img src="../img/rally-logo.png" width="250" />
>> -                    <img src="../img/galvanize.png" width="250"/>
>> -                    <img src="../img/you-here.png" width="250" />
>> +                    <a href="http://www.codeforamerica.org/"><img src="../img/CfA_logo_lg.png" width="250" /></a>
>> +                    <a href="https://www.rallydev.com/"><img src="../img/rally-logo.png" width="250" /></a>
>> +                    <a href="http://www.galvanize.it/"><img src="../img/galvanize.png" width="250"/></a>
>> +                    <a href="#"><img src="../img/you-here.png" width="250" /></a>
> Should this really be a link?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/codefordenver/code-for-denver-site/pull/47/files#r24874944